web/src/features/fba/components/infoPanel/AdvisoryText.tsx

```diff
@@ -201,6 +202,11 @@ const AdvisoryText = ({
 
     return (
       <>
+        {selectedFireZoneUnit && (
+          <Typography data-testid="fire-zone-unit-bulletin" sx={{ whiteSpace: 'pre-wrap' }}>
+            {zoneTitle}
```

Comment:
What do you think about matching the font color and font weight for the zone title in the advisory text with the zone title in the infographic section?

Reply:
You mean like the same color as the advisory/warning status? I'm open to it.

Reply:
This is what I was thinking:
![image](https://github.com/user-attachments/assets/f66f509a-c981-4cf1-b53b-b4eacf08cb83)